docker/README.md

```diff
@@ -108,6 +108,8 @@ While the SFTPGo container is in graceful shutdown mode waiting for the last con
 
 If no connections are active or `SFTPGO_GRACE_TIME=0` (default value if unset) the container will shutdown immediately.
 
+**Warning**: simply using `docker stop some-sftpgo` will not give you the full graceful shutdown period. To use SFTPGO's graceful period, then run this command: `docker exec some-sftpgo kill 1` which will trigger the graceful shutdown of SFTPGO. If you need to wait for the container to stop, then run `docker exec some-sftpgo kill 1 ; docker wait some-sftpgo`
```

Comment:
please use `docker kill` to send a signal to a running container instead of executing the `kill` command within the container (not tested but the above command will likely fail with distroless images). 

Some other notes:

- replace **Warning** with `:warning:`
- in sentences like "To use SFTPGO's" please replace SFTPGO with SFTPGo

maybe you start the warning with something like this (English is not my native launguage)

A graceful shutdown is triggered by an interrupt signal so ....

Thanks!

Reply:
Sure. In January after xmas